default/stringtables/en.txt

```diff
@@ -4628,6 +4628,14 @@ Species Traits
 SPECIES_TRAITS_TEXT
 Non-exhaustive list of Species Traits:
 
+SYSTEM_SUPERIORITY_TITLE
+System Superiority
+
+SYSTEM_SUPERIORITY_TEXT
+'''When an armed fleet set to aggressive mode arrives at a system unoccupied by hostile fleets, it establishes system superiority. Once it has been established, supply is interrupted for hostile empires, industry of hostile planets can only be used on the planet it is generated on, and ground troop regeneration is stopped. Also, all starlane entries in the occupied system become guarded by the blockading force, so if hostile forces arrive, they can only exit by the starlane entry by which they arrived.
+If two hostile fleets arrive at an unoccupied system at the same time, no system superiority is established until only one side remains and the other one has either been destroyed or retreated. Until then, both empires will maintain supply for their empire.
```

Comment:
Right, yes, the previous supply mechanics are now superseded in that respect and so the clause
```both empires will maintain supply for their empire.```

should be changed to something like 
```neither such empire will establish a blockade against each other, although they will both establish blockades against all other empires.```

Also we should probably have at least have some acknowledgement of supply competition, at least a link to an article about supply in general which includes supply competition. 

I have not examined the supply competition code but my experience ingame leads me to believe that supply competition and blockades are two separate potential blocks, and the results I see seem to indicate to me that the blockade effect is taken into consideration first, before supply competition is assessed.  So it seems to me the rest of the supply blockade discussion here is still OK, and that it would be most clear to have a separate supply article which would talk about supply meters and how supply propagation can be blocked either by blockades or by supply competition.  And then either here or in the general article it should mention how a blockade can allow supply to propagate across an opposing supply source.  That seems very closely related to the supply competition idea, so I would probably leave it for the general supply article.